### PR TITLE
Changed Source.type to String

### PR DESCRIPTION
**Pull request by wintercodeNKJJ**
Closes issue #194

**file:**
`src > types > elbwalker.dts`  (line 122)

**Changes:**
```diff
interface Source {
+   type: string;
-    type: SourceType;
    ...
  }
```
**file:**
`src > elbwalker.ts`  (line 335)

```diff
    const source = {
+    type: 'web',
-     type: IElbwalker.SourceType.Web,
      id: window.location.href,
      previous_id: document.referrer,
    };
```
**Test Results:**
image

### CODE REVIEW

File: src/elbwalker.ts [hunk 1]
```diff
@@ -332,7 +332,7 @@ function Elbwalker(
     const timing = Math.round((performance.now() - config.timing) / 10) / 100;
     const id = `${timestamp}-${config.group}-${config.count}`;
     const source = {
-      type: IElbwalker.SourceType.Web,
+      type: 'web',
```
Comment (alexanderkirtzel) [inline src/elbwalker.ts:335]: I'd prefer to have a centralized place for const values.
The current status with const enum in types isn't good, too (see #193 )
But right now we'd have unused `const enum SourceType` in `elbwalker.d.ts`.

Why not just switching `Web = 1` to `Web = 'web'`?

File: src/types/elbwalker.d.ts [hunk 2]
```diff
@@ -119,7 +119,7 @@ export namespace IElbwalker {
   }
 
   interface Source {
-    type: SourceType;
+    type: string;
```
Comment (alexanderkirtzel) [inline src/types/elbwalker.d.ts:122]: What about the `elbwalker.test.ts` isn't there a test to check that it has to be 'web'? Before it was `1`. It's either not working, or we need to add/edit a test for this

**Review by alexanderkirtzel — CHANGES_REQUESTED** [review]
Please keep in mind to also update the overall `README.md` with the new structured event format.
And is it possible to merge in `develop` branch?

File: src/elbwalker.ts (hunk 1 above)
Comment (wintercodeNKJJ) [reply src/elbwalker.ts:335]: this is what I obtain as result when doing that
```bash
Types of property 'type' are incompatible.
        Type 'string' is not assignable to type 'SourceType.Web'
```

**Pull request merged**